app/controllers/sessions_controller.rb

```diff
@@ -9,11 +9,7 @@ def create
     @user = User.find_by(username: params[:session][:username])
     if @user && @user.authenticate(params[:session][:password])
       session[:user_id] = @user.id
-      if session[:login_redirect] == "/cart"
```

Comment:
we'll need to recreate this functionality for redirecting back to the item page when they log in from an item's show page